guides/common/modules/con_managing-alternate-content-sources.adoc

```diff
@@ -1,15 +1,17 @@
 :_mod-docs-content-type: CONCEPT
 
-[id="Managing_Alternate_Content_Sources_{context}"]
+[id="managing-alternate-content-sources"]
 = Managing alternate content sources
 
-Alternate content sources define alternate paths to download content during synchronization.
+[role="_abstract"]
+Alternate content sources (ACS) define alternate paths to download content during synchronization.
 The content itself is downloaded from the alternate content source, while the metadata is downloaded from the {ProjectServer} or the upstream URL, depending on the configuration.
-You can use alternate content source to speed up synchronization if the content is located on the local filesystem or on a nearby network.
-You can set up alternate content sources for {ProjectServer} and {SmartProxy}.
+You can use alternate content sources to speed up synchronization if the content is located on the local filesystem or on a nearby network.
+You can set up alternate content sources for {ProjectServer} and {SmartProxyServers}.
+
```

Comment:
Needs style ACK: Can I have two paragraphs here? If not, I need to delete this empty line again.